weed/shell/command_volume_server_evacuate.go

```diff
@@ -47,6 +48,7 @@ func (c *commandVolumeServerEvacuate) Do(args []string, commandEnv *CommandEnv,
 
 	vsEvacuateCommand := flag.NewFlagSet(c.Name(), flag.ContinueOnError)
 	volumeServer := vsEvacuateCommand.String("node", "", "<host>:<port> of the volume server")
+	volumeRack := vsEvacuateCommand.String("rack", "", "rack for then volume servers")
```

Comment:
typo

Is the rack of the source or target volume server?

Reply:
> Is the rack of the source or target volume server?

source